src/portal/components/NodeActionBtn.vue

```diff
@@ -75,13 +75,14 @@ export default class NodeActionBtn extends Vue {
     );
     return await getRentStatus(this.$api, this.nodeId, currentTwinID);
   }
-  reserveNode(nodeId: string) {
+  reserveNode(nodeId: string, solutionProviderID:string) {
```

Comment:
I think we will need to add a default value for it, maybe 0 for now
```suggestion
  reserveNode(nodeId: string, solutionProviderID: number) {
```